crates/services/compression/src/service.rs

```diff
@@ -102,7 +116,7 @@ where
             .commit()
             .map_err(crate::errors::CompressionError::FailedToCommitTransaction)?;
 
-        Ok(())
+        Ok(compressed_block)
```

Comment:
Before you commit to the database, you can get the size of the compressed block from the `storage_tx`, because we already serialized it there. And here we can already return the size

Reply:
addressed in 2157152ae3

Reply:
extra fixed in [d695a07](https://github.com/FuelLabs/fuel-core/pull/2920/commits/d695a07a83865252ff8100d9cf7fa149df442319)